tests/misc/llext-edk/pytest/test_edk.py

```diff
@@ -64,3 +72,36 @@ def test_edk(dut: DeviceAdapter):
 
             # Check if the extension was built
             assert os.path.exists(Path(tempdir_extension) / "build/extension.llext")
+
+            # Can we run it? First, rebuild the application, now including the extension
+            # build directory in the include path, so that the application can find the
+            # extension code.
+            logger.debug(f"Running application with extension in {tempdir_extension} - west build")
+            command = [
+                "west",
+                "build",
+                "-b",
+                raw_dut.device_config.platform,
+                "--build-dir",
+                raw_dut.device_config.build_dir,
```

Comment:
Yeah, that's sad. Your comment about `west build` (and #75291) is fair. While there's only one EDK test now, I can imagine more, all depending on the existence of the EDK. If we had a way to fine control west builds from pytest, it would be nice - here, I need two runs of west, one to build the EDK that will be used for the extension (so, with a different target), and another to build the application that uses the extension (needing an extra parameter).